integration_tests/tests/steps/node_steps.rs

```diff
@@ -851,18 +851,19 @@ async fn generate_block_with_2_as_single_request_coinbases(world: &mut TariWorld
     }
     assert_eq!(coinbase_kernel_count, 1);
     assert_eq!(coinbase_utxo_count, 2);
-    let mut num_6154266700 = 0;
-    let mut num_12308533398 = 0;
+    let mut num_6154272109 = 0;
+    let mut num_12308544218 = 0;
     for output in body.outputs() {
-        if output.minimum_value_promise.as_u64() == 6154266700 {
-            num_6154266700 += 1;
+        if output.minimum_value_promise.as_u64() == 6154272109 {
+            num_6154272109 += 1;
         }
-        if output.minimum_value_promise.as_u64() == 12308533398 {
-            num_12308533398 += 1;
+        if output.minimum_value_promise.as_u64() == 12308544218 {
+            num_12308544218 += 1;
         }
     }
```

Comment:
These values should equal the total block reward, should it not? We could assert that.

Reply:
It is checked below by the fact that the block is accepted as valid